components/sendgrid/actions/search-contacts/search-contacts.js

```diff
@@ -0,0 +1,31 @@
+const sendgrid = require("../../sendgrid.app");
+
+module.exports = {
+  key: "sendgrid-search-contacts",
+  name: "Search Contacts",
+  description: "Searches contacts with a SGQL query.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    sendgrid,
+    query: {
+      type: "string",
+      label: "Query",
+      description:
+        "The query field accepts valid SGQL for searching for a contact. Only the first 50 contacts will be returned. If the query takes longer than 20 seconds a 408 Request Timeout will be returned.",
+    },
+  },
+  async run() {
+    const validate = require("validate.js");
```

Comment:
Can you move this to the top, like the other `require`s?

Reply:
Sure, I think I already started to move the `require`s to the top in other action files.